README_CICD_INTEGRATION.md

```diff
@@ -82,14 +86,23 @@ As a build artifact a docker image is wanted versioned with bb-gitversion.
 The example builds the docker image with the Dockerfile located in `docker/Dockerfile` and creates a tag generated by a previous step running `bb-gitversion`.
 
 Adapt the last section in the `pipeline.yaml` to match your needs. The `image-((bb_name))` will be called `image-bb-example` and is needed to define the target docker image. The name and destination is configured in `config.yaml`. This image section does not need to be modified. But you might to modify the job called `Create docker image for ((bb_name)) and push it`.
-The only two parts you need to modify are in the last task `put: image-((bb_name))` 
+The only parts you need to modify are in the two last jobs:
+
+-   `name: Create docker image for ((bb_name)) and push it`
+-   `name: Create latest tag for docker image for ((bb_name)) and push it`
+
+There you need to modify these sections
 
 -   `build` (docker build context) 
 -   `dockerfile` (location of your Dockerfile). 
 
-If you want your image to be tagged as `latest` keep the value at `true`, otherwise switch to `false`.
+The pipeline can create two resulting docker images:
+
+1.  For your specified target branch with versioning information, either with tag `FullSemVer` for `master` or `InformationalVersion` for every other branch. 
```

Comment:
`FullSemVer` and `InformationalVersion` arent introduced  here. Maybe, adding a link to https://gitversion.net/docs/more-info/variables helps.

Reply:
done